src/xenia/gpu/gl4/command_processor.cc

```diff
@@ -335,7 +335,7 @@ bool CommandProcessor::SetupGL() {
       "    }\n"
       "    EmitVertex();\n"
       "    EndPrimitive();\n"
-      "  } else {\n"
+      "  } else if (gl_in[0].gl_Position.y == gl_in[1].gl_Position.y) {\n"
```

Comment:
Yeah, it might be possible. If it falls through nothing will be drawn.

Depends on which will get our attention faster, no rects or corrupt rects on screen